nvm.sh

```diff
@@ -1314,6 +1314,10 @@ nvm() {
         NVM_NODE_PREFIX="$(nvm_node_prefix)"
         PROVIDED_VERSION="$2"
         case "_$PROVIDED_VERSION" in
+          "_default")
+            # Special case for performance, as this is run on every shell startup
+            VERSION="$(nvm_alias default 2> /dev/null)"
+          ;;
```

Comment:
This actually turns out not to work correctly when the default alias is itself a symbol like "iojs" that needs to be resolved further.